styles.css

```diff
@@ -0,0 +1,865 @@
+  padding-left: 340px;
+  padding-bottom: 25px;
+  color: var(--metallic-bronze);
+}
+
+.container-6 h2 {
+  width: 134px;
+  height: 32px;
+  font-family: var(--font-family);
+  font-weight: var(--font-weight);
+  font-style: var(--font-style);
+  font-size: 20px;
+  line-height: 32px;
+  text-align: center;
+  color: var(--tahiti-gold);
+  margin: 10px 20px 5px 25px;
+  padding-left: 490px;
+}
+
+.container-6 {
+  background-image: url(./img/background-feedback.png);
+  background-repeat: no-repeat;
+  background-position: bottom center;
+}
+
+.for-background-2 {
+  background-image: url(./icon/people-3.svg);
+  background-repeat: no-repeat;
+  background-position: bottom 35px center;
+}
+
+.container-7 {
+  background-image: url(./img/background-for-email.png);
+  background-repeat: no-repeat;
+}
+
+.container-7 .heiding {
+  padding-top: 120px;
+}
+
+.container-7 .text {
+  max-width: 682px;
+  margin-left: 230px;
+}
+
+#email {
+  color: var(--pickled-bean);
+  border-radius: 70px;
+  padding-left: 30px;
+}
+
+input {
+  border: none;
+  outline: none;
+  width: 770px;
+  height: 80px;
+  background-color: var(--rose-white);
+  position: relative;
+}
+
+input::-webkit-input-placeholder {
+  color: var(--shadow);
+}
+
+input::-moz-placeholder {
+  color: var(--shadow);
+}
+
+form {
+  box-sizing: border-box;
+  text-align: center;
+  margin-left: 150px;
+  padding: 0 10px 80px 30px;
+ }
+
+form .button {
+  position: relative;
+  left: -200px;
+}
+
+.container-8 {
+  display: flex;
+  padding-top: 40px;
+}
+
+.icon img {
+  padding-bottom: 20px;
+}
+
+.icon p {
+  width: 234px;
+  font-family: var(--font-family);
+  font-size: var(--font-size);
+  font-style: var(--font-style);
+  line-height: var(--line-height);
+  font-weight: 400;
+  color: var(--metallic-bronze);
+  padding-bottom: 20px;
+  margin-right: 20px;
+}
+
+.icons a {
+  border: none;
+  outline: none;
+  text-decoration: none;
+}
+
+.icons img {
+  width: 21px;
+  margin-right: 10px;
+}
+
+.categories {
+  padding-left: 80px;
+  padding-right: 80px;
+}
+
+.categories h2 {
+  width: 102px;
+  font-family: var(--font-family);
+  font-weight: var(--font-weight);
+  font-style: var(--font-style);
+  font-size: 20px;
+  line-height: 32px;
+  color: var(--metallic-bronze);
+  padding-bottom: 20px;
+}
+
+.categories a {
+  display: block;
+  text-decoration: none;
+  color: var(--metallic-bronze);
+  margin-bottom: 10px;
+}
+
+.information {
+  padding-left: 80px;
+  padding-right: 80px;
+}
+
+.information h2 {
+  width: 174px;
+  font-family: var(--font-family);
+  font-weight: var(--font-weight);
+  font-style: var(--font-style);
+  font-size: 20px;
+  line-height: 32px;
+  color: var(--metallic-bronze);
+  padding-bottom: 20px;
+}
+
+.information a {
+  display: block;
+  text-decoration: none;
+  color: var(--metallic-bronze);
+  margin-bottom: 10px;
+}
+
+.contact {
+  padding-left: 80px;
+}
+
+.contact h2 {
+  width: 100px;
+  font-family: var(--font-family);
```

Comment:
you can add to a body font family and remove it everywhere

Reply:
fixed